docs/api/composables.md

```diff
@@ -179,19 +179,17 @@ watch(carRef, ({ model }) => {
 })
 ```
 
-## useTresContext (former useTres)
-
-This composable aims to provide access to the state model which contains the default renderer, camera, scene, and other useful properties.
+## useTresContext
+<!-- IMPROVE -->
+This composable aims to provide access to the state model which contains multiple useful properties.
 
 ```ts
-const { camera, renderer } = useTresContext()
+const { camera, renderer, camera, cameras } = useTresContext()
```

Comment:
And here we have two cameras, the rest looks OK to me

Reply:
camera is the active camera and camera**s** is the array of existing cameras.

Reply:
Yeah but looks, it says, camera, render CAMERA, cameras, is that ok?